soc/arm/nordic_nrf/nrf52/dts.fixup

```diff
@@ -124,6 +124,15 @@
 #define CONFIG_WDT_NRF_IRQ		NORDIC_NRF_WATCHDOG_40010000_IRQ_WDT
 #define CONFIG_WDT_NRF_IRQ_PRI		NORDIC_NRF_WATCHDOG_40010000_IRQ_WDT_PRIORITY
 
+#if defined(NORDIC_NRF_CC310_5002A000_BASE_ADDRESS)
+#define CONFIG_CC310_CTL_BASE_ADDR	NORDIC_NRF_CC310_5002A000_BASE_ADDRESS
+#define CONFIG_CC310_CTL_NAME		NORDIC_NRF_CC310_5002A000_LABEL
+#define CONFIG_CC310_BASE_ADDR		ARM_CRYPTOCELL_310_5002B000_BASE_ADDRESS
```

Comment:
Is this base address going to be used anywhere?

Reply:
If we want HW accelerated crypto then this address must be used.

For example [nrf_cc310 library from nRF5 SDK](http://infocenter.nordicsemi.com/topic/com.nordic.infocenter.sdk5.v15.2.0/lib_cryptocell.html?cp=4_0_0_3_54) is accessing registers from this address.
If we want to have something similar in Zephyr then it should use this address too. That's why this define is important.